legal-api/src/legal_api/resources/v2/business/business_filings/business_filings.py

```diff
@@ -872,12 +889,7 @@ def create_invoice(business: Business,  # pylint: disable=too-many-locals,too-ma
     def set_effective_date(business: Business, filing: Filing):
         """Set the effective date of the Filing."""
         filing_type = filing.filing_json['filing']['header']['name']
-        if filing_type in CoreFiling.NEW_BUSINESS_FILING_TYPES:
-            if fe_date := filing.filing_json['filing']['header'].get('futureEffectiveDate'):
-                filing.effective_date = datetime.datetime.fromisoformat(fe_date)
-                filing.save()
-
-        elif business.legal_type != Business.LegalTypes.COOP.value and filing_type == 'changeOfAddress':
```

Comment:
we don't have a header called `futureEffectiveDate`, `effectiveDate` is handled in line no 610